### PR TITLE
US 1583733 Add missing language IDs - 52

### PR DESCRIPTION
**Pull request by v-thpra**
US 1583733 - Fix CATS P1 issues by adding missing language IDs to code blocks.
- Used "console" for command-line tool examples. 
- Used "console" for sample program output displayed on the console.
- Used "text" for example of text input to a sample program.
- Used "csv" for example of csv input to a sample program.

### CODE REVIEW

File: docs/visual-basic/programming-guide/concepts/async/async-return-types.md [hunk 1]
```diff
@@ -270,7 +270,7 @@ End Sub
   
      The following output should appear.
```
Comment (Youssef1313) [inline docs/visual-basic/programming-guide/concepts/async/async-return-types.md]: The `.` may be replaced with `:`

File: docs/visual-basic/programming-guide/concepts/async/cancel-an-async-task-or-a-list-of-tasks.md [hunk 2]
```diff
@@ -121,14 +121,14 @@ Then add the following changes to the MainWindow.xaml.vb file of that project.
 
 5. If you don’t cancel the program, it produces the following output.
```
Comment (Youssef1313) [inline docs/visual-basic/programming-guide/concepts/async/cancel-an-async-task-or-a-list-of-tasks.md]: Same here, `:` seems to be more appropriate.

File: docs/visual-basic/programming-guide/concepts/async/cancel-an-async-task-or-a-list-of-tasks.md [hunk 3]
```diff
@@ -121,14 +121,14 @@ Then add the following changes to the MainWindow.xaml.vb file of that project.
...
 
-    ```
+    ```console
     Ready to download.
     Length of the downloaded string: 158125.
     ```
 
     If you choose the **Cancel** button before the program finishes downloading the content, the program produces the following output.
```
Comment (Youssef1313) [inline docs/visual-basic/programming-guide/concepts/async/cancel-an-async-task-or-a-list-of-tasks.md]: Same comment as before.

File: docs/visual-basic/programming-guide/concepts/async/cancel-an-async-task-or-a-list-of-tasks.md [hunk 4]
```diff
@@ -218,7 +218,7 @@ To extend the example yourself, step by step, follow the instructions in the "Do
 
 5. If you don’t cancel the program, it produces the following output.
```
Comment (Youssef1313) [inline docs/visual-basic/programming-guide/concepts/async/cancel-an-async-task-or-a-list-of-tasks.md]: Same comment as before.

File: docs/visual-basic/programming-guide/concepts/async/cancel-async-tasks-after-a-period-of-time.md [hunk 5]
```diff
@@ -67,7 +67,7 @@ End Sub
   
  Run the program several times to verify that the output might show output for all websites, no websites, or some web sites. The following output is a sample.
```
Comment (Youssef1313) [inline docs/visual-basic/programming-guide/concepts/async/cancel-async-tasks-after-a-period-of-time.md]: Same comment as before.

File: docs/visual-basic/programming-guide/concepts/async/control-flow-in-async-programs.md [hunk 6]
```diff
@@ -58,7 +58,7 @@ End Class
 
 Each of the labeled locations, "ONE" through "SIX," displays information about the current state of the program. The following output is produced.
```
Comment (Youssef1313) [inline docs/visual-basic/programming-guide/concepts/async/control-flow-in-async-programs.md]: Same comment as before.

File: docs/visual-basic/programming-guide/concepts/async/control-flow-in-async-programs.md [hunk 7]
```diff
@@ -216,7 +216,7 @@ To run the project, perform the following steps:
 
     The following output should appear.
```
Comment (Youssef1313) [inline docs/visual-basic/programming-guide/concepts/async/control-flow-in-async-programs.md]: Same comment as before.

File: docs/visual-basic/programming-guide/concepts/async/control-flow-in-async-programs.md [hunk 8]
```diff
@@ -303,7 +303,7 @@ Dim getLengthTask As Task(Of Integer) = AccessTheWebAsync()
 
 As in `AccessTheWebAsync`, `startButton_Click` can continue with work that doesn’t depend on the results of the asynchronous task (`getLengthTask`) until the task is awaited. The following output lines represent that work.
```
Comment (Youssef1313) [inline docs/visual-basic/programming-guide/concepts/async/control-flow-in-async-programs.md]: Same comment as before.

File: docs/visual-basic/programming-guide/concepts/async/control-flow-in-async-programs.md [hunk 9]
```diff
@@ -323,7 +323,7 @@ In the following illustration, the arrows show the flow of control from the awai
 
 When `client.GetStringAsync` signals that it’s complete, processing in `AccessTheWebAsync` is released from suspension and can continue past the await statement. The following lines of output represent the resumption of processing.
```
Comment (Youssef1313) [inline docs/visual-basic/programming-guide/concepts/async/control-flow-in-async-programs.md]: Same comment as before.

File: docs/visual-basic/programming-guide/concepts/async/handling-reentrancy-in-async-apps.md [hunk 10]
```diff
@@ -241,7 +241,7 @@ End Function
 
 If you choose the **Start** button several times while this app is running, it should produce results that resemble the following output.
```
Comment (Youssef1313) [inline docs/visual-basic/programming-guide/concepts/async/handling-reentrancy-in-async-apps.md]: Same comment as before.

File: docs/visual-basic/programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await.md [hunk 11]
```diff
@@ -211,7 +211,7 @@ Visual Studio 2012 or later must be installed on your computer. For more informa
 
     Output that resembles the following list should appear.
```
Comment (Youssef1313) [inline docs/visual-basic/programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await.md]: Same comment as before.

File: docs/visual-basic/programming-guide/concepts/async/async-return-types.md (hunk 1 above)
Comment (v-thpra) [reply docs/visual-basic/programming-guide/concepts/async/async-return-types.md]: fixed.

**Comment by v-thpra** [comment]
@Youssef1313 Thanks for catching those.

**Review by rpetrusha — APPROVED** [review]
Thanks, @v-thepet. I'll merge your PR now.

**Pull request merged**